soc/silabs/silabs_siwx917/siwg917/soc.c

```diff
@@ -40,6 +41,16 @@ int silabs_siwx917_init(void)
 }
 SYS_INIT(silabs_siwx917_init, PRE_KERNEL_1, 0);
 
+/* Co-processor will use value stored in IVT to store its stack. See also linker.ld.
+ *
+ * FIXME: We can't use Z_ISR_DECLARE() to declare this entry
+ * FIXME: Allow to configure size of buffer
+ */
+uint8_t __aligned(4) siwx917_coprocessor_stack[10 * 1024];
+static Z_DECL_ALIGN(struct _isr_list) Z_GENERIC_SECTION(.intList) __used __isr_siwx917_coprocessor_stack_irq = {
```

Comment:
Can this be split into multiple lines? Compliance is failing due to the long line.